templates/main.template.yaml

```diff
@@ -1003,6 +1003,52 @@ Conditions:
     - !Condition install_nfw_app
     - !Condition create_nfw_source
 
+  cloudtrail_validation_bucket: !And
```

Comment:
I think we should do all validations either only in child templates (recommended) or in main template not in both places. passing of these conditions and parameters makes code very complex

Reply:
need both places, because when the user deploys the master template it will use validation in common. When a user deploys a child template exam: Securiy Hub it will use validation in the child template, just like SumoLogicHelperFunction